src/components/DefinitionList/DefinitionList.scss

```
@@ -4,10 +4,24 @@
 $block: '.#{variables.$ns}definition-list';
 
 #{$block} {
-    $class: &;
-
     margin: 0;
 
+    &__title {
+        margin-block-end: var(--g-spacing-3);
+
+        &:not(:first-of-type) {
+            margin-block-start: var(--g-spacing-5);
+        }
+    }
+
+    #{$block}__item:not(#{$block}__item_grouped) + #{$block}__subheader {
```

```suggestion
    #{$block}__item + #{$block}__subheader {
```

Let's have same margin before the group - 5sp, now it's 3sp

---

This rule seems to be redundant, because margin before group is defined with [this](https://github.com/gravity-ui/components/blob/31ebef368dcf4e0ec8f30fac6e0fc8812abab00b/src/components/DefinitionList/DefinitionList.scss#L9) rule.
Or you want to have 5sp margin between group title and its items?..